src/pages/ItemsList/CheckButton/CheckBox.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const CheckBox = ({ id, name, checkCheckBox, themeData, handlerThemeInfo }) => {
+  const [saveThemeId, setSaveCheckThemeId] = useState(id);
+
+  const saveCheckTheme = ({ target }) => {
+    setSaveCheckThemeId(target.id);
+    handlerThemeInfo(saveThemeId, target.name);
+  };
+
+  return (
+    <CheckBoxContainer htmlFor={id}>
+      <CheckBoxInput
+        name={name}
+        id={id}
+        checked={themeData.includes(String(id)) ? true : false}
```

Comment:
삼항연산자에서 불리안 값을 이용할 경우 어떻게 단축시킬 수 있을까요?!
```suggestion
        checked={themeData.includes(String(id))}
```

Reply:
...? 아니!? boolean 값일 때는 생략이 가능한 건가요...??
정말 생략해도 잘 동작하네요? 신기합니다... 원리는 이해가 잘 가지는 않는데
0이 아니면, 값이 존재한다면 truthy 한 값임을 이용한건가요?